pkg/sources/reconciler/azureeventgridsource/errors.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2021 TriggerMesh Inc.
```

Comment:
The year should be 2022, it also happens in the others files

Reply:
I'll do it, but it's absolutely not mandatory to apply the Apache v2 license. Other Apache projects don't bump the year in their headers. Maintainers even explicitly ask contributors not to do it (I don't know the reason).

Reply:
Uh didn't know that, good to know!

Reply:
Done in 65ef49410449935a8f2cf261dc5472d9af79e737